src/WordPress/Blueprints/ResourceManager/Resolver/FilesystemResourceResolver.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace WordPress\Blueprints\ResourceManager\Resolver;
+
+use WordPress\Blueprints\Model\Builder\FilesystemResourceBuilder;
+use WordPress\Blueprints\Model\DataClass\ResourceDefinitionInterface;
+use WordPress\Blueprints\Model\DataClass\FilesystemResource;
+
+class FilesystemResourceResolver implements ResourceResolverInterface
+{
+
+    public function parseUrl(string $url): ?ResourceDefinitionInterface
+    {
+        if (! str_starts_with($url, 'file://')) {
+            return null;
+        }
+
+//    TODO does this Builder exist?
```

Comment:
Nope, good catch. It's just `FilesystemResource` now

Reply:
Cool. Review if the proposed fix is suitable, please.